admin/client/src/legacy/GridField.js

```diff
@@ -133,21 +134,52 @@ $.entwine('ss', function($) {
 		}
 	});
 
+  $('.grid-field .action.action_import:button').entwine({
+    onclick: function(e) {
+      e.preventDefault();
+
+      var backdrop = $('.modal-backdrop');
+
+      if(backdrop.length < 1) {
+        backdrop = $('<div class="modal-backdrop fade in"></div>');
+        $('body').append(backdrop);
+      } else {
+        backdrop.addClass('fade in').fadeIn();
+      }
+
+      $(this.data('target')).addClass('in');
+
+      // workaround until GridField rewritten into react.
+      var self = this;
+
+      $(this.data('target')).find('[data-dismiss]').on('click', function() {
+        backdrop.fadeOut(function() {
+          $(this).removeClass('in');
```

Comment:
use `backdrop` rather than `$(this)`, makes it easier to predict what's happening at a glance.